### ISSUE TITLE
Parameter 'extract-images' expects type <symbol> but got <string>

### ISSUE THREAD

**Issue by phenix-factory** [djcb/mu#1661]
Hello !

I keep getting this error when opening a mail, the mail in the "loading" state, nothing is showed : 

```
error in process filter: mu4e-error-handler: Error 1: parameter 'extract-images' expects type <symbol> but got <string> (line was:'(view :docid 89757 :msgid nil :extract-images "t" :decrypt nil :verify t)')
error in process filter: Error 1: parameter 'extract-images' expects type <symbol> but got <string> (line was:'(view :docid 89757 :msgid nil :extract-images "t" :decrypt nil :verify t)')
```
I m using mbsync and Spacemacs.
I don't know what to do. But i don't have the issue using v1.2.

**Comment by djcb** [comment]
Would help if you'd follow the issue-template and specify versions you're using....

Anyway, can you reproduce with the git version?

**Comment by phenix-factory** [comment]
I can confirm that it's fixed in 1.4.2.

Sorry for the template, won't happen again.
